projects/igniteui-angular/src/lib/combo/combo-dropdown.component.ts

```diff
@@ -366,9 +372,16 @@ export class IgxComboDropDownComponent extends IgxDropDownBase {
     /**
      * @hidden
      */
-    onToggleClosing() {
+    onToggleClosing(e: CancelableEventArgs) {
         this.combo.searchValue = '';
-        super.onToggleClosing();
+        const eventArgs = { cancel: false };
+        this.onClosing.emit(eventArgs);
+        e.cancel = eventArgs.cancel;
+        if (eventArgs.cancel) {
+            return;
+        }
+
+        super.onToggleClosing(e);
```

Comment:
With the `super` call in place, the combo emitts the closing event twice when e.cancel === false

Reply:
yeah, just saw this - fixed